net-misc/netsed/netsed-1.3-r1.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 2022 Gentoo Authors
```

Comment:
Why change this copyright header? Everything looks good otherwise.

Reply:
I did not, pkgdev changed it aoutmatically 😕

Reply:
I did not 😕 I find sometimes pkgdev did it during pkgdev commit